### PR TITLE
Promote from BLAIS5-3953 to main

### PR DESCRIPTION
**Pull request by social-surveys-blaise-concourse**
Auto generated by concourse:

Promote from BLAIS5-3953 to main

### CODE REVIEW

**Comment by elthorne** [comment]
This PR includes:

* Targeted links to Data Delivery, NISRA, and BTS playbooks for corresponding Slack alerts
* Silences all sandbox alerts except Training
* Refactoring of some typos and unused imports

File: lib/filters/sandbox_filter.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+from lib.log_processor import ProcessedLogEntry
+
+
+def is_formal_environment(log_name):
```
Comment (JAWilliamsONS) [inline lib/filters/sandbox_filter.py:4]: could this be '_is_formal_environment'? Can't see it being used anywhere else and they are '_is_totalmobile_alert' etc. below (in slack/slack_message.py)

File: tests/lib/filters/test_sandbox_filter.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
+import pytest
+import datetime
+import dataclasses
+
+from lib.log_processor import ProcessedLogEntry
+from lib.filters.sandbox_filter import sandbox_filter
+
+
+@pytest.fixture()
```
Comment (JAWilliamsONS) [inline tests/lib/filters/test_sandbox_filter.py:9]: Is it worth moving the fixtures for all these filters into a conftest and renaming them and importing them 🤷 genuine question... if you are bored

**Review by JAWilliamsONS — APPROVED** [review]

File: tests/lib/filters/test_sandbox_filter.py (hunk 2 above)
Comment (JAWilliamsONS) [reply tests/lib/filters/test_sandbox_filter.py:9]: https://stackoverflow.com/questions/73191533/using-conftest-py-vs-importing-fixtures-from-dedicate-modules 
It's a philosophical question... how bored are you?

File: lib/filters/sandbox_filter.py (hunk 1 above)
Comment (elthorne) [reply lib/filters/sandbox_filter.py]: Agreed, good sir! 🧐

File: tests/lib/filters/test_sandbox_filter.py (hunk 2 above)
Comment (elthorne) [reply tests/lib/filters/test_sandbox_filter.py:9]: Not for me, thanks! 😆

**Pull request merged**